### PR TITLE
Extend JIT functionality

### PR DESCRIPTION
**Pull request by kostorr**
- Utilize URIs from repo
- Enable JIT for QC node gworkflows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,8 +195,41 @@ odcEndpoint: yet-another-host-ib:22334
...
+    - Alternatively, a custom DPL command can be provided through the "Advanced
+      Configuration" panel, which will **take precedence** over the workflow
+      normally selected through the interface. See the next subsection for details.
+2) Consul payloads contained in the DPL command
+    - These are parsed from the provided DPL command string and Consul is queried
+      regarding their version to ensure freshness.
+3) JIT-specific env vars
+    - These are expected on the deployment's Consul instance under
+      `o2/components/aliecs/[defaults|vars]/jit_env_vars`
+4) The O2 version
```
Comment (knopers8) [inline README.md:229]: The QC version is not queried?

**Review by knopers8 — COMMENTED** [review]
Thanks a lot, i just have a few comments to the documentation

File: README.md (hunk 1 above)
Comment (kostorr) [reply README.md:229]: No, is it also necessary? I can easily add it.

Comment (kostorr) [reply README.md]: Done 🚀

Comment (knopers8) [reply README.md]: Yes, thanks

**Pull request merged**